examples/#06 - Exploring Integrations.ipynb

```diff
@@ -0,0 +1,6963 @@
+{
```

Comment:
A review está na conversação, com o comentário e o número da célula, da forma que está notebook.

Reply:
[134] Pode carregar o dataframe direo com `pymove.read_csv`

Reply:
[7] Tentar remover esse `warning`

Reply:
[143 , 12] Pode vetorizar essa operação. Primeiro filtra so os elementos que são pontos. Depois pega as coordenadas x e y deles.
```python
POIs[POIs['geometry'].type == 'Point']
POIs['lat'] = POIs['geometry'].x
POIs['lon'] = POIs['geometry'].y
```

Reply:
[Heading 7] Modificar a terminologia: `It integrates a normal dataframe with a CVP (?) with information from Points of Interest. In this example, we will assign random datetime values to some POIs to simulate the operation.`

Reply:
[131] Remover essa célula. Melhor adicionar um comentário no início do notebook, logo após ao primeiro cabeçalho, informando que são necessárias as libs `osmnx` e `geopandas`, com a instrução para instalar.

Reply:
[135] Importar o `geopandas` com os outros imports, no início do notebook.

Reply:
Pode plotar a trajetória e os pontos de interesse.
```python
m = folium.plot_trajectories_with_folium(move_df, color=['gray'])
folium.add_poi_folium(POIs, slice_tags=['amenity'], base_map=m)
```